### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/realopinsight/actions/runs/5682494915#summary-15401001697

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/realopinsight/actions/runs/5682495132#summary-15401002264

### CODE REVIEW

File: core/src/K8sHelper.cpp [hunk 1]
```diff
@@ -450,7 +450,7 @@ std::tuple<int,  std::string, std::string> K8sHelper::extractStateInfo(const QJs
 
   // default state is "waiting"
   // See Kubernetes documentation: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#containerstate-v1-core
-  return std::make_tuple(ngrt4n::K8sPending, "", QObject::tr("container is wating").toStdString());
+  return std::make_tuple(ngrt4n::K8sPending, "", QObject::tr("container is waiting").toStdString());
```
Comment (jsoref) [inline core/src/K8sHelper.cpp:453]: This probably breaks translations.

There is an `fr` translation. For simplicity, I opted to not touch that file, and am hoping your translation process will deal w/ the change, but I can adjust it if requested.

File: core/src/utilsCore.hpp [hunk 2]
```diff
@@ -164,7 +164,7 @@ bool findNode(const NodeListT &nodes, const QString &nodeId, NodeListT::const_it
 
 void setCheckOnError(int status, const QString &msg, CheckT &invalidCheck);
 
-StringPairT splitDataPointInfo(const QString &info); /* return <[sourcei:]hostaddr, checkid> */
+StringPairT splitDataPointInfo(const QString &info); /* return <[source:]hostaddr, checkid> */
```
Comment (jsoref) [inline core/src/utilsCore.hpp:167]: ??

File: docs/deployment-on-kubernetes.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ helm upgrade --namespace monitoring --install realopinsight manifests/helm/realo
 ```
 
 ## In-cluster access to service
-On success, the deployment does create a Kubernetes **ClusterIP** service named `realopinsiht`.
+On success, the deployment does create a Kubernetes **ClusterIP** service named `realopinsight`.
```
Comment (jsoref) [inline docs/deployment-on-kubernetes.md:31]: brand

**Comment by rchakode** [comment]
Hi @jsoref,
Thank you for P, we will review it.

File: README.md [hunk 4]
```diff
@@ -28,8 +28,8 @@ Key features:
 ![](./images/banners/screenshots.png)
 
 # Getting Started
-  * [Deployment on Kubernetes](./docs/deployement-on-kubernetes.md)
-  * [Deployment on Docker](./docs/deployement-on-deployment-on-docker.md)
+  * [Deployment on Kubernetes](./docs/deployment-on-kubernetes.md)
+  * [Deployment on Docker](./docs/deployment-on-deployment-on-docker.md)
```
[the remark this answers is not in the document]
Comment (rchakode) [reply README.md:32]: I think this breaks the 2nd link

[the remark this answers is not in the document]
Comment (jsoref) [reply README.md]: The second link was already very broken the actual file is https://github.com/rchakode/realopinsight/blob/main/docs/deployment-on-docker.md. I've fixed the link.

File: core/src/K8sHelper.cpp (hunk 1 above)
Comment (rchakode) [reply core/src/K8sHelper.cpp:453]: Yes, I'll update the fr translation.

File: core/src/utilsCore.hpp (hunk 2 above)
Comment (rchakode) [reply core/src/utilsCore.hpp:167]: This is a comment for code readability, the index of the source is relevant.

**Comment by rchakode** [comment]
Dear @jsoref 
Thank you again for this huge contribution.
I'll merge it and review the `fr` translation afterhand.

**Pull request merged**